_docs/install/prerequisites/firewalls.md

```diff
@@ -11,80 +11,74 @@ StorageOS requires a few ports to be open in order to function correctly. These
 
 | Port Number | TCP/UDP   | Use                    |
 |:-----------:|:---------:|:---------------------- |
+| 5701        | tcp       | gRPC                   |
+| 5702        | tcp       | Prometheus             |
+| 5703        | tcp       | DirectFS               |
+| 5704        | tcp       | Dataplane health check |
 | 5705        | tcp       | REST API               |
 | 5706        | tcp       | ETCD service           |
 | 5707        | tcp       | ETCD service           |
-| 8001        | tcp       | Dataplane health check |
-| 4222        | tcp       | NATS service           |
-| 4223        | tcp       | NATS service           |
-| 8222        | tcp       | NATS service           |
-| 13701       | tcp & udp | Gossip service         |
-| 17100       | tcp       | Direct FS              |
-| 17102       | tcp       | Direct FS              |
+| 5708        | tcp       | NATS service           |
+| 5709        | tcp       | NATS service           |
+| 5710        | tcp       | NATS service           |
+| 5711        | tcp & udp | Gossip service         |
 
 StorageOS also uses random ports to dial-out to these ports on other sotrageOS nodes. For this reason, outgoing traffic should be enabled.
 
 
 ## Firewalls and VPS providers
 
-Some VPS providers (such as Digital Ocean) have some default firewall rules set which must be updated to allow StorageOS to run.
+Some VPS providers (such as Digital Ocean) have some default firewall rules set
+which must be updated to allow StorageOS to run.
 
-As many of these hosting providers seem to be using `UFW` as a firewall, here is a list of commands to configure `UFW` to allow StorageOS:
+To configure firewall rules using `UFW` commands:
 
 ```bash
 ufw default allow outgoing
+ufw allow 5701/tcp
+ufw allow 5702/tcp
+ufw allow 5703/tcp
+ufw allow 5704/tcp
 ufw allow 5705/tcp
 ufw allow 5706/tcp
 ufw allow 5707/tcp
-ufw allow 8001/tcp
-ufw allow 4222/tcp
-ufw allow 4223/tcp
-ufw allow 8222/tcp
-ufw allow 13701/tcp
-ufw allow 13701/udp
-ufw allow 17100/tcp
-ufw allow 17102/tcp
+ufw allow 5708/tcp
+ufw allow 5709/tcp
+ufw allow 5710/tcp
+ufw allow 5711/tcp
```

Comment:
and udp:

```
ufw allow 5711/udp
```

(I haven't tested this though)

Reply:
Done.